### DISCUSSION TITLE
demo_hpm compilation flag

### DISCUSSION THREAD

**Discussion by mmcheraghi** [stnolting/neorv32#1129] [General]
Hi
Is it necessary to mention _zicsr and _zihpm in MARCH while compiling demo_hpm program?
I compiled it without those mentioned and it worked.

**Comment by stnolting** [comment]
Hey @mahdi259!

`_zicsr` is required for the CSR-access instructions. However, `_zihpm` is not required as this ISA extension does not contain any additional instructions (it is just for identification).

**Comment by mmcheraghi** [comment]
Thanks a lot
